### PR TITLE
OKTA-554534 Part 2 event hook filtering guide

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Initial draft of a simple event hook filter example
- **Is this PR related to a Monolith release?** Removed from 2023.04.0 release. Release TBD

### Resolves:

* [OKTA-554534](https://oktainc.atlassian.net/browse/OKTA-554534)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/index.md](https://github.com/okta/okta-developer-docs/blob/deab50778d3da8f5e602364097ceff2a4a6bca4b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/9ccf196d-20d3-40fe-adab-bed7f0ce365b) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/deab50778d3da8f5e602364097ceff2a4a6bca4b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [73](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3a0cc0cf-6a43-40fc-afb5-9d0501d4f5ba) | 50  | 79  | 66  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.1f9dc832-d92a-4ffa-806f-5fd383850b38?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/index.md](https://github.com/okta/okta-developer-docs/blob/ab43922db5512bc9497824836ace525fef5e3b45/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1ea50dc1-47d0-4931-9901-893483dc84a5) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/ab43922db5512bc9497824836ace525fef5e3b45/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [73](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6b3806d2-472e-4066-8696-6acfaf6f0a7f) | 51  | 79  | 66  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.9bb829ae-f2dd-4f9f-a427-6333544bff23?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/index.md](https://github.com/okta/okta-developer-docs/blob/07d6d4031bdcac8fec3c9c0a5aace28348cf5441/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ca5573cd-2870-4826-a7e5-e762a83d8b6c) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/07d6d4031bdcac8fec3c9c0a5aace28348cf5441/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0499c86c-a19d-4b2e-8d1c-0fd813a13a63) | 74  | 74  | 100  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.d420add7-b52b-43bd-becb-b4e2911708d7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/index.md](https://github.com/okta/okta-developer-docs/blob/918a98924f0087bb5de4b0222a7441080f9913d0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/36df6b99-2613-4adc-a5d9-e8540deb3748) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/918a98924f0087bb5de4b0222a7441080f9913d0/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8d94f29a-694c-4d2e-99ba-47f1bb3dd907) | 74  | 74  | 100  | 98  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.5f5a4b7a-83ec-451f-b5e0-00d9715b8eec?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/index.md](https://github.com/okta/okta-developer-docs/blob/f2730f55b6a7a8c90c8cbd504bd70ed29ead1730/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c94ccf09-54dc-4b1e-a45d-12aab31bf0b1) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/f2730f55b6a7a8c90c8cbd504bd70ed29ead1730/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/77e7e4e1-8993-4adf-b15c-485358ca56f2) | 74  | 74  | 100  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/f2730f55b6a7a8c90c8cbd504bd70ed29ead1730/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [67](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b5401dc2-ad87-4082-96a1-c29c32f0127e) | 38  | 53  | 74  | 85  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.18866655-aac9-4424-b630-d4e253d7b292?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+**Sample code**
+
+* [Okta Event Hook with Filtering](https://glitch.com/~okta-event-hook-with-filtering)
+
+---
+
+## About event hook filters
+
+Event hook filters reduce the amount of event hook calls to your external service endpoint. The filter triggers hooks only for events based on the logic that you define. You can implement one or more filters to your event hook and apply them when creating a hook or later. For further information, see [Event hooks](/docs/concepts/event-hooks/#which-events-are-eligible).
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md:34]:
```suggestion
Event hook filters reduce the amount of event hook calls to your external service endpoint. The filter triggers hooks only for events based on the logic that you define. You can implement one or more filters to your event hook and apply them when creating a hook or apply them later. See [Event hooks](/docs/concepts/event-hooks/#which-events-are-eligible).
```

**Review by susanharper-okta — APPROVED** [review]
couple nits - approving to not hold you up.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/index.md](https://github.com/okta/okta-developer-docs/blob/9d3a0e4fa3cf12227151df32cf09284407301f7f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Findex.md) | [100](https://okta.acrolinx.cloud/api/v1/checking/scorecards/08c03880-1ee8-4139-99c3-670592119d18) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/event-hook-filtering/main/index.md](https://github.com/okta/okta-developer-docs/blob/9d3a0e4fa3cf12227151df32cf09284407301f7f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fevent-hook-filtering%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/bf9c80d7-6470-4b4f-afea-54094c6889b4) | 69  | 79  | 100  | 98  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/index.md](https://github.com/okta/okta-developer-docs/blob/9d3a0e4fa3cf12227151df32cf09284407301f7f/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Findex.md) | [66](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3880aef6-04ae-4026-9c80-adf472e41745) | 38  | 53  | 74  | 85  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.1d22503e-dc2f-441b-abc9-efabd2ad56b7?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**